Userland/Libraries/LibWasm/AbstractMachine/Validator.h

```diff
@@ -251,11 +277,40 @@ class Validator {
         template<typename T, typename U, typename V>
         static ValidationError out_of_bounds(StringView name, V value, T min, U max) { return String::formatted("Value {} for {} is out of bounds ({},{})", value, name, min, max); }
 
-#if WASM_VALIDATOR_DEBUG
-        static ValidationError invalid_stack_state(SourceLocation location = SourceLocation::current());
-#else
-        static ValidationError invalid_stack_state();
-#endif
+        template<typename... Expected>
+        static ValidationError invalid_stack_state(Stack const& stack, Tuple<Expected...> expected, SourceLocation location = SourceLocation::current())
```

Comment:
Does the location argument here need a `[[maybe_unused]]` for when WASM_VALIDATOR_DEBUG is not defined?

Reply:
nah, it's leftover from when I was using `#if`, will remove